static/js/components/SearchBox.jsx

```diff
@@ -242,9 +242,9 @@ const SearchBox = ({ sources }) => {
           <div style={{ display: "inline-block" }}>
             <i>
               Displaying&nbsp;
-              {sources.sourceNumberingStart}
+              {sources.numberingStart}
```

Comment:
These are renamed to make it easier to use the same query results pagination function for both the scanning page & source list GET requests